unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -469,21 +469,21 @@ public void SetEntityParent(string entityId, string parentId)
 
             if (me != null)
             {
-                if (parentId == "PlayerEntityReference")
+                if (parentId == "FirstPersonCameraEntityReference")
                 {
-                    me.SetParent(DCLCharacterController.i.playerReference);
+                    me.SetParent(DCLCharacterController.i.firstPersonCameraReference);
```

Comment:
I remember that we defined that the `Player` would represent the Avatar in third person but the camera in first person. This applies here as well? So attaching something to `FirstPersonCameraEntityReference` would attach to the camera in FPS mode but the avatar in TPS?

Reply:
Hey Brian!

Yes, the functionality is still the same. When you do `entity.setParent(Attachable.FIRST_PERSON_CAMERA)`, then the entity will be attached to the avatar, and rotate with the camera.

If the user then switched to third person camera, it will still be attached to the avatar. It was just a name change, because the community found `Player` vs `Avatar Position` confusing